### PR TITLE
Replace broken images

### LINKED ISSUE #27: Replace broken images with a placeholder image
Get Image from https://placeholder.com/

This will fix:
capture

### PR DESCRIPTION
**Pull request by bruce-wayne99**
Fixed #27 
replaced broken images with placeholder image of size 50x50

### CODE REVIEW

**Comment by bruce-wayne99** [comment]
broken-image

File: src/app/components/album/album.component.html [hunk 1]
```diff
@@ -24,7 +24,10 @@ <h2>Tracks</h2>
 	<div class="row">
 		<div class="col-md-12">
 			<div class="well" *ngFor="let track of pagedItems">
-				<img src="{{album.image[3]['#text']}}" height="50" alt="{{track.name}}">
+				<a href="https://placeholder.com">
```
Comment (jajodiaraghav) [inline src/app/components/album/album.component.html]: Why are we linking the images to https://placeholder.com ?

File: src/app/components/artist/artist.component.html [hunk 2]
```diff
@@ -27,7 +27,10 @@ <h2>Albums</h2>
 	<div class="row">
 		<div class="col-md-12">
 			<div class="well" *ngFor="let album of pagedItems">
-				<img src="{{album.image[3]['#text']}}" height="50" alt="{{album.name}}">
+				<a href="https://placeholder.com">
```
Comment (jajodiaraghav) [inline src/app/components/artist/artist.component.html]: Why are we linking the images to https://placeholder.com ?

**Comment by bruce-wayne99** [comment]
Removed linking of images to placeholder.com

**Comment by jajodiaraghav** [comment]
Thanks @bruce-wayne99 

Advice:
I would recommend you to go through this link: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Pull request merged**